### PR TITLE
Add info about submissions to working group page

### PR DESCRIPTION
**Pull request by calvin-lau-sig7**

### CODE REVIEW

**Comment by calvin-lau-sig7** [comment]
Added @EoinShaughnessy as a reviewer for this update.

We probably need a subject matter expert on the rules and governance of the working group (Community Manager?). I'll raise in standup.

File: src/community/design-system-working-group/index.md.njk [hunk 1]
```diff
@@ -14,6 +14,15 @@ The working group:
 - makes recommendations to help contributors improve their work
 - advocates for the needs of colleagues across government
 
+The working group reviews:
+
+- new patterns and components
+- changes to components that will easily be recognised
```
Comment (EoinShaughnessy) [inline src/community/design-system-working-group/index.md.njk:20]: I know we mean it's the changes that will be recognised, but someone might think it's the components we're referring to. Would "component changes"work?

Can we say who will recognise the changes and make this active? (So, "component changes that users will easily recognise," or similar.)

File: src/community/design-system-working-group/index.md.njk [hunk 2]
```diff
@@ -14,6 +14,15 @@ The working group:
...
+- changes to components that will affect when they should be used
```
Comment (EoinShaughnessy) [inline src/community/design-system-working-group/index.md.njk:21]:
```suggestion
- changes that will affect when users should use components
```

File: src/community/design-system-working-group/index.md.njk [hunk 3]
```diff
@@ -14,6 +14,15 @@ The working group:
...
+- content that will affect how users make services - and how services are assessed
```
Comment (EoinShaughnessy) [inline src/community/design-system-working-group/index.md.njk:22]: Can we say who assesses services and make the last clause active?

**Review by EoinShaughnessy — COMMENTED** [review]
Content looks good! Added a few small comments.

File: src/community/design-system-working-group/index.md.njk (hunk 1 above)
Comment (calvin-lau-sig7) [inline src/community/design-system-working-group/index.md.njk:20]:
```suggestion
- component changes that users will easily recognise
```

File: src/community/design-system-working-group/index.md.njk (hunk 3 above)
Comment (calvin-lau-sig7) [reply src/community/design-system-working-group/index.md.njk:21]:
```suggestion
- content that will affect how users make services that meet the Service Standard
```
Slightly different meaning, but what do you think @EoinShaughnessy?

Comment (EoinShaughnessy) [reply src/community/design-system-working-group/index.md.njk:21]: @calvin-lau-sig7 That sounds really good. When you put it like that, it feels like the previous wording was splitting out things unnecessarily.

Comment (calvin-lau-sig7) [reply src/community/design-system-working-group/index.md.njk:21]: Cheers @EoinShaughnessy. Yes looks a bit obvious now doesn't it?

Comment (EoinShaughnessy) [reply src/community/design-system-working-group/index.md.njk]: @calvin-lau-sig7 It always does after someone else has done the hard work 😀

**Review by EoinShaughnessy — APPROVED** [review]
Looks good!

File: src/community/design-system-working-group/index.md.njk [hunk 4]
```diff
@@ -14,6 +14,14 @@ The working group:
 - makes recommendations to help contributors improve their work
 - advocates for the needs of colleagues across government
 
+The working group reviews:
+
+- component changes that users will easily recognise
```
Comment (calvin-lau-sig7) [inline src/community/design-system-working-group/index.md.njk:19]:
```suggestion
- new patterns and components
- component changes that users will easily recognise
```

Comment (calvin-lau-sig7) [inline src/community/design-system-working-group/index.md.njk:19]:
```suggestion
The working group reviews:

- new patterns and components
- component changes that users will easily recognise
```

File: src/community/design-system-working-group/index.md.njk [hunk 5]
```diff
@@ -14,6 +14,14 @@ The working group:
...
+- changes that will affect when users should use components
+- content that will affect how users make services and meet the Service Standard
+
+The working group does not review minor design or content changes. For example, updated guidance on using a component.
```
Comment (calvin-lau-sig7) [inline src/community/design-system-working-group/index.md.njk:23]:
```suggestion
The working group does not review minor design or content changes, such as updated guidance on using a component.
```

**Comment by calvin-lau-sig7** [comment]
Hi @trang-erskine, when you get a chance can you take a look at the text I added in 'Files changed'? 

Once you're happy this is ready to merge - however will need some dev help as there's a failed test and the commits need squashing.

**Review by trang-erskine — COMMENTED** [review]
This looks great.

**Pull request merged**